### DISCUSSION TITLE
Who painted the Sistine Chapel ceiling?

### DISCUSSION THREAD

**Discussion by narayan954** [ParanoidUser/yolo#229] [Galaxy Trivia] [answered]
Hint: He was also a famous sculptor

**Comment by ParanoidUser** [comment]
**Extra hint / answer** 😁 Great question, @narayan954. Thank you for your active participation!

![teenage-mutant-ninja-turtles-michelangelo](https://user-images.githubusercontent.com/5120290/223739706-b5dba3f9-545e-44c0-bb8f-d4da1a1b2aa0.gif)

**Comment by LauraBigoni** [comment, marked as answer]
Ofc it was **Michelangelo Buonarroti**!

**Reply by narayan954** [reply]
And its a correct answer, congrats!

**Reply by LauraBigoni** [reply]
As an Italian i couldnt get it wrong xD
